service/linkAccount.js

```diff
@@ -6,24 +6,22 @@ const tokenCollection = 'token';
 
 let isTokenAlreadyUsed = (token) => {
     return new promise(function (resolve, reject) {
-        dbHelper.dbClient.collection(tokenCollection).findOne(token, {}, function (err, match) {
-            if (err) {
-                reject(err);
-            }
-            else {
-                if (match) {
-                    resolve(true);
+        dbHelper.dbClient.collection(tokenCollection).findOne(
```

Comment:
to make it easier to read, we always like to chain method calls with a new line: 

```
dbHelper
.dbClient
.collection(tokenCollection)
.findOne()
...
```

Try to use this pattern other places too :)